### PR TITLE
Implement Marika's legal suggestions for EFNY form builder 

### PR DESCRIPTION
**Pull request by sraby**
This PR adds and changes a few elements to our Eviction Free NY declaration builder, according to Marika's legal suggestions:
1. It switches the order and expands the "what is hardship" accordion questions on the hardship checklist page
2. It adds an accordion to the index number page with images of where to find the index number itself
3. It adds an accordion to our landlord name/address autofill page, explaining more details about the implications of overriding landlord info from HPD

### CODE REVIEW

**Comment by sraby** [comment]
[ch6028]

**Comment by sraby** [comment]
[ch6029] [ch6031]

File: frontend/lib/ui/accordion.tsx [hunk 1]
```diff
@@ -10,12 +10,17 @@ export const Accordion = (props: {
...
 }) => {
   const extraClassName = props.extraClassName ?? "jf-space-below-2rem";
 
   return (
     <div className={`jf-accordion-item ${extraClassName}`}>
-      <details className={`has-text-left ${extraClassName}`}>
+      <details
+        className={`has-text-left ${extraClassName}`}
+        open={props.isExpanded}
+      >
```
Comment (toolness) [inline frontend/lib/ui/accordion.tsx:23]: oh wow SUPER DOPE, excellent work sir.  I had wondered how you were going to do this part, I didn't realize there was a built-in `open` attribute for `<details>`!

**Review by toolness — APPROVED** [review]
NOICE

**Pull request merged**